wordpress-importer.php

```diff
@@ -988,7 +988,19 @@ function fetch_remote_file( $url, $post ) {
 			return new WP_Error( 'upload_dir_error', $upload['error'] );
 
 		// fetch the remote url and write it to the placeholder file
-		$headers = wp_get_http( $url, $upload['file'] );
+		$response = wp_remote_get( $url, array( 'timeout' => 60) );
+		$headers = wp_remote_retrieve_headers( $response );
+		if ( is_wp_error() | '200' !=  wp_remote_retrieve_response_code( $response ) ) {
```

Comment:
The argument for `is_wp_error()` is missing. Should also use a strict check there.